### PR TITLE
stock_move_description [10.0]

### PR DESCRIPTION
**Pull request by AbdulrhmanGad**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why are you overwriting the authors/contributors? You should add yourself to them, but not erasing previous ones.

File: stock_move_description/__manifest__.py [hunk 1]
```diff
@@ -20,10 +20,12 @@
...
 {
     'name': "Stock move description",
-    'version': '8.0.1.0.0',
-    'category': 'Warehouse Management',
-    'author': "Agile Business Group, Odoo Community Association (OCA)",
-    'website': 'http://www.agilebg.com',
+    'version': '10.0.1.0.0',
+        'category': 'Warehouse Management',
+    'author': "Agile Business Group, Odoo Community Association (OCA)"
+    "abdulrhman Mohammed ",
```
Comment (rousseldenis) [inline stock_move_description/__manifest__.py]: @AbdulrhmanGad What @pedrobaeza said that you must not change the original author but put yourself in Contributors part in the README file.

File: stock_move_description/stock_config_settings.py [hunk 2]
```diff
@@ -19,18 +19,21 @@
...
+    group_use_product_description_per_stock_move = fields.Boolean(
+        "Allow using only the product description on the stock moves",
+        help="Allows you to use only product description on the "
+        "stock moves."
+        )
+
+
+    @api.multi
+    def set_group_product_desc_values(self):
+        return self.env['ir.values'].sudo().set_default('stock.config.settings',
```
Comment (rousseldenis) [inline stock_move_description/stock_config_settings.py]: @AbdulrhmanGad Why changing a group type field behaviour to this ?

File: stock_move_description/stock_move.py [hunk 3]
```diff
@@ -19,44 +19,42 @@
...
-class StockMove(orm.Model):
+class StockMove(models.Model):
     _inherit = "stock.move"
+    name = fields.Text(string="Description", required=False, )
 
-    _columns = {
-        'name': fields.text('Description'),
-    }
 
     def onchange_product_id(
```
Comment (rousseldenis) [inline stock_move_description/stock_move.py]: @AbdulrhmanGad If it is an onchange function, it must be decorated with the good api.onchange().

File: stock_move_description/stock_move.py [hunk 4]
```diff
@@ -19,44 +19,42 @@
...
         res = super(StockMove, self).onchange_product_id(
-            cr, uid, ids, prod_id=prod_id, loc_id=loc_id,
+             prod_id=prod_id, loc_id=loc_id,
             loc_dest_id=loc_dest_id, partner_id=partner_id
         )
         if prod_id:
-            user = self.pool.get('res.users').browse(
-                cr, uid, uid, context=context)
+            user = self.env['res.users'].browse(
+                 context=context)
```
Comment (rousseldenis) [inline stock_move_description/stock_move.py]: @AbdulrhmanGad Context is not passed through function arguments. If you want current user, you can use self.env.user

File: stock_move_description/stock_move_view.xml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<odoo>
     <data>
```
Comment (rousseldenis) [inline stock_move_description/stock_move_view.xml:3]: @AbdulrhmanGad You can remove data section, odoo is sufficient

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review

File: stock_move_description/__manifest__.py [hunk 6]
```diff
@@ -20,10 +20,10 @@
 ##############################################################################
 {
     'name': "Stock move description",
-    'version': '8.0.1.0.0',
-    'category': 'Warehouse Management',
+    'version': '10.0.1.0.0',
+        'category': 'Warehouse Management',
```
Comment (gurneyalex) [inline stock_move_description/__manifest__.py:24]: fix indentation

File: stock_move_description/__manifest__.py [hunk 7]
```diff
@@ -20,10 +20,10 @@
...
     'author': "Agile Business Group, Odoo Community Association (OCA)",
-    'website': 'http://www.agilebg.com',
+    'website': 'http://www.agilebg.com' ,
```
Comment (gurneyalex) [inline stock_move_description/__manifest__.py:26]: remove space

File: stock_move_description/stock_move.py [hunk 8]
```diff
@@ -19,44 +19,40 @@
...
-    _columns = {
-        'name': fields.text('Description'),
-    }
-
+    @api.onchange('product_id')
     def onchange_product_id(
-            self, cr, uid, ids, prod_id=False, loc_id=False,
-            loc_dest_id=False, partner_id=False, context=None
+            self, prod_id=False, loc_id=False,
+            loc_dest_id=False, partner_id=False
```
Comment (gurneyalex) [inline stock_move_description/stock_move.py:32]: this is not the correct prototype for an onchange method in the new API

File: stock_move_description/stock_move.py [hunk 9]
```diff
@@ -19,44 +19,40 @@
...
     ):
-        context = dict(context or {})
+
         res = super(StockMove, self).onchange_product_id(
-            cr, uid, ids, prod_id=prod_id, loc_id=loc_id,
+             prod_id=prod_id, loc_id=loc_id,
             loc_dest_id=loc_dest_id, partner_id=partner_id
```
Comment (gurneyalex) [inline stock_move_description/stock_move.py:37]: in v10 the prototype of this method has changed

**Review by gurneyalex — CHANGES_REQUESTED** [review]
Also: remove .pyc files

**Comment by fuentes73** [comment]
@AbdulrhmanGad are you planning to attend the comments? We're at Spain OCA code sprint and if you want, we can do it.

**Comment by rousseldenis** [comment]
@AbdulrhmanGad Any new on this?

**Comment by rousseldenis** [comment]
@AbdulrhmanGad

File: stock_move_description/__manifest__.py (hunk 7 above)
Comment (nikul-serpentcs) [reply stock_move_description/__manifest__.py:26]: here URL like `https://github.com/OCA/stock-logistics-workflow`

**Comment by rousseldenis** [comment]
Closing this as no answer. Please feel free to reopen it if you want to finish the work

**Pull request closed without merging**